addons/core/tests/integration/components/image-test.js

```diff
@@ -8,7 +8,7 @@ module('Integration | Component | Image', function (hooks) {
 
   test('it renders', async function (assert) {
     assert.expect(1);
-    await render(hbs`<Image @name="resource-creation" />`);
+    await render(hbs`<Image @name="logo-app" />`);
```

Comment:
This was unrelated to the topic of this PR.
But deleting the `resource-creation` imaged from core, reveal that we were using such image to test the `<image>` component which is not related to `resource-creation`. SInce we just need to test it renders an image I changed to the app logo which most likely will never be deleted.